Tests/BackendIntegrationTests/Constants.swift

```diff
@@ -10,12 +10,12 @@ import Foundation
 
 enum Constants {
 
-    static let apiKey = "REVENUECAT_API_KEY"
-    static let loadShedderApiKey = "REVENUECAT_LOAD_SHEDDER_API_KEY"
-    static let customEntitlementComputationApiKey = "REVENUECAT_CUSTOM_ENTITLEMENT_COMPUTATION_API_KEY"
+    static let apiKey = "appl_gyJuOxwYgimOegOmmobUGlRoJGV"
```

Comment:
Oops, you might want to setup the pre-commit hook: https://github.com/RevenueCat/purchases-ios/blob/main/scripts/pre-commit.sh